mh/constants.py

```diff
@@ -232,6 +232,59 @@ def slot(item, qty):
 
     return data
 
+LAYER_CHAT_COLORS = (0xbb3385ff, 0xffffffff, 0xffffffff)
+
+QUEST_FLAG_UNK = 0b1000  # It really is unknown, it put a question mark icon on the monster/item box icon
```

Comment:
A nested class can be used (see `PatServerType` down below)? (enum doesn't exist in Python2)

```python
class Quest:
    class Flag:
        ARENA = (...)

    class Location:
        NONE = (...)
```

Moreover, the whole quest support is incomplete and should probably be done in another PR.

Reply:
This change has been dropped from this PR but I would take in consideration your comment.